src/netcore/SnWebApplicationWithIdentityAndMemDb/SnWebApplicationWithIdentityAndMemDb/Startup.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.UI;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.EntityFrameworkCore;
+using SnWebApplicationWithIdentity.Data;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using SenseNet.Configuration;
+using SenseNet.ContentRepository;
+using SenseNet.ContentRepository.Security;
+using SenseNet.ContentRepository.Storage.Data.MsSqlClient;
+using SenseNet.Diagnostics;
+using SenseNet.Identity.Experimental;
+using SenseNet.OData;
+using SenseNet.Search.Lucene29;
+using SenseNet.Security.EFCSecurityStore;
+using SenseNet.Services.Core;
+
+namespace SnWebApplicationWithIdentity
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration, IHostEnvironment env)
+        {
+            Configuration = configuration;
+            Environment = env;
+        }
+
+        private IHostEnvironment Environment { get; }
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // [sensenet]: Identity store
+            services.AddSenseNetIdentity()
+                .AddDefaultUI();
+
+            services.AddRazorPages();            
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IHostApplicationLifetime applicationLifetime)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+                app.UseDatabaseErrorPage();
+            }
+            else
+            {
+                app.UseExceptionHandler("/Home/Error");
+                // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+                app.UseHsts();
+            }
+            app.UseHttpsRedirection();
+            app.UseStaticFiles();
+
+            app.UseRouting();
+
+            // [sensenet]: Authentication
+            app.UseSenseNetAuthentication();
+
+            app.UseAuthorization();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllerRoute(
+                    name: "default",
+                    pattern: "{controller=Home}/{action=Index}/{id?}");
+                endpoints.MapRazorPages();
+            });
+
+            // [sensenet]: OData
+            app.UseSenseNetOdata();
+
+        }
+
+        private static RepositoryInstance _repository;
+        private void StartRepository()
```

Comment:
I think we do not need this method, because in `Program.cs` we use the built-in method for starting an in-mem repo.